include/picongpu/simulation_defines/param/fieldBackground.param

```diff
@@ -49,8 +49,8 @@ namespace picongpu
         {
             /* example: periodicity of 20 microns ( = 2.0e-5 m) */
             constexpr float_64 period_SI(20.0e-6);
-            /* calculate cells -> SI -> m to microns*/
-            const float_64 y_SI = cellIdx.y() * SI::CELL_HEIGHT_SI * 1.0e6;
+            /* calculate cells -> SI [m] */
+            const float_64 y_SI = cellIdx.y() * SI::CELL_HEIGHT_SI;
```

Comment:
offline discussion: the var `y_SI` was wrongly named (should have been `y_mu` or similar) and is then used below wrongly which causes the mismatch between intended and implemented period in this example